packages/main/src/plugin/install/extension-installer.ts

```diff
@@ -145,37 +163,49 @@ export class ExtensionInstaller {
     const tmpFolderPath = path.join(os.tmpdir(), `/tmp/${imageNameWithoutSpecialChars}-tmp`);
 
     // final folder
-    const finalFolderPath = path.join(this.extensionLoader.getPluginsDirectory(), imageNameWithoutSpecialChars);
+    const finalFolderPath = path.join(unpackedFolder, imageNameWithoutSpecialChars);
 
     // grab all extensions
-    const extensions = await this.extensionLoader.listExtensions();
+    if (isPDExtension) {
```

Comment:
Looks like we don"t check for Docker Desktop extensions ?

Reply:
no because it'll require some fields that we don't have yet

Reply:
(and today it's overriding the extension, you can already install an existing Docker Desktop extension, it's reinstalling it)